src/components/PersonsParentLink.tsx

```diff
@@ -0,0 +1,33 @@
+import classNames from 'classnames';
+import { Person } from '../types';
+import { Link, useSearchParams } from 'react-router-dom';
+
+type Props = {
+  personName: string;
+  peopleList: Person[];
+};
+
+export const PersonsParentLink: React.FC<Props> = ({
+  personName,
+  peopleList,
+}) => {
+  const [searchParams] = useSearchParams();
+  const targetPerson = peopleList.find(human => human.name === personName);
+
+  if (targetPerson) {
+    const { name, slug, sex } = targetPerson;
+
+    return (
+      <Link
+        to={`/people/${slug}?${searchParams}`}
+        className={classNames({
+          'has-text-danger': sex === 'f',
+        })}
+      >
+        {name}
+      </Link>
+    );
+  }
+
+  return undefined;
```

Comment:
Let's change a little bit this logic 
```suggestion
  if (!targetPerson) {
  	return undefined;
  }
  
    const { name, slug, sex } = targetPerson;

    return (
      <Link
        to={`/people/${slug}?${searchParams}`}
        className={classNames({
          'has-text-danger': sex === 'f',
        })}
      >
        {name}
      </Link>
    );
```